src/features/navigation/RootNavigator/Header/AccessibleTabBar.web.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import styled from 'styled-components/native'
+
+import { useCurrentRoute } from 'features/navigation/helpers'
+import { navigateFromRef } from 'features/navigation/navigationRef'
+import { getTabNavConfig } from 'features/navigation/TabBar/helpers'
+import { mapTabRouteToBicolorIcon } from 'features/navigation/TabBar/mapTabRouteToBicolorIcon'
+import { TabBarComponent } from 'features/navigation/TabBar/TabBarComponent'
+import { MainContainer, TabBarContainer } from 'features/navigation/TabBar/TabBarContainer'
+import { useTabNavigationContext } from 'features/navigation/TabBar/TabNavigationStateContext'
+import { TabRouteName } from 'features/navigation/TabBar/types'
+
+export const AccessibleTabBar = () => {
+  const { tabRoutes } = useTabNavigationContext()
+  const currentRoute = useCurrentRoute()
+
+  if (currentRoute && currentRoute.name !== 'TabNavigator') return null
+
+  return (
+    <AccessibleTabBarContainer>
```

Comment:
Je pense que tu peux remplacer ça par `<Link to={{ screen: route.name, params; undefined }} />`